src/optimizer.f90

```diff
@@ -980,17 +980,24 @@ subroutine modhes(env, modh, natoms, xyz, chg, Hess, pr)
       call env%error("internal error in model hessian!", source)
       return
    case(p_modh_old)
-     if (pr) write(env%unit,'(a)') "Using Lindh-Hessian (1995)"
+     if (pr) write(env%unit,'(x,a)') "Using Lindh-Hessian (1995)"
      call ddvopt(xyz, natoms, Hess, chg, modh%s6)
    case(p_modh_lindh_d2)
-     if (pr) write(env%unit,'(a)') "Using Lindh-Hessian"
+     if (pr) write(env%unit,'(x,a)') "Using Lindh-Hessian"
      call mh_lindh_d2(xyz, natoms, Hess, chg, modh)
    case(p_modh_lindh)
-     if (pr) write(env%unit,'(a)') "Using Lindh-Hessian (2007)"
+     if (pr) write(env%unit,'(x,a)') "Using Lindh-Hessian (2007)"
      call mh_lindh(xyz, natoms, Hess, chg, modh)
    case(p_modh_swart)
-     if (pr) write(env%unit,'(a)') "Using Swart-Hessian"
+     if (pr) write(env%unit,'(x,a)') "Using Swart-Hessian"
```

Comment:
What is the motivation for this change? Also the `x` specifier without a width is an error with GNU compilers and debug settings.

Reply:
has been changed accordingly.